2-mandatory/3-financial-times.js

```diff
@@ -14,7 +21,12 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    
+  return allArticleTitles.reduce((shortest, sentence) {
+      
+    return shortest.length < sentence.length ? shortest : sentence;
```

Comment:
`sentence.length` will give you the number of characters in the title, however this question is asking for the fewest number words.